.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: required
```

Comment:
Good question! The [Travis docs](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Node.js-v4-(or-io.js-v3)-compiler-requirements) state that to compile NodeJS v4 or later a newer C++ compiler is needed. 
That compiler is currently available in images that also allow [sudo rights in your build](https://docs.travis-ci.com/user/trusty-ci-environment).
So while we don't need `sudo` explicitly we need the build image that offers it.
Does that make sense?